asset-transfer-basic/rest-api-typescript/src/server.ts

```diff
@@ -0,0 +1,67 @@
+import helmet from 'helmet';
+import { StatusCodes, getReasonPhrase } from 'http-status-codes';
+import express, { NextFunction, Request, Response } from 'express';
+import pino from 'pino';
+import pinoMiddleware from 'pino-http';
+
+const logger = pino({
+  level: process.env.LOG_LEVEL || 'info',
+});
+
+const app = express();
+const { BAD_REQUEST, INTERNAL_SERVER_ERROR, OK } = StatusCodes;
+
+app.use(pinoMiddleware({
```

Comment:
think I'll steal this

Reply:
I stole it from https://github.com/nodeshift/nodejs-reference-architecture !